src/actions/src.cr

```diff
@@ -1,26 +1,113 @@
+require "../ext/path"
```

Comment:
It will likely be needed for other actions as well. Can be moved to `actions.cr`
It will also avoid a backward reference.